### PR TITLE
Rename `install flux` to `enable repo`

### LINKED ISSUE #1417: Rename `install flux` to `enable repo`
In order to unify the UI of the gitops commands `eksctl install flux` should be renamed to `eksctl enable repo`. That also makes the command's interface more agnostic to the underlying technology used for gitops.

### PR DESCRIPTION
**Pull request by martina-if**
closes #1417

This branch contains a commit from another PR that needs to be merged first: #1420

- [x] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, and `examples` directory)
- [x] Manually tested

### CODE REVIEW

File: pkg/nodebootstrap/assets.go [hunk 1]
```diff
@@ -1,16 +1,16 @@
-// Code generated by go-bindata. DO NOT EDIT.
```
Comment (rndstr) [inline pkg/nodebootstrap/assets.go]: this reverts it to the old bindata version, no longer required after #1420

File: integration/creategetdelete_test.go [hunk 2]
```diff
@@ -124,7 +124,7 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 			})
 		})
 
-		Context("and configuring Flux for gitops", func() {
+		Context("and enabling a gitops repo", func() {
```
Comment (marccarre) [inline integration/creategetdelete_test.go]: Nit: what does "_and enabling a gitops repo_" mean? 🤔 
Do we want to say "_enabling a repo for gitops_"? "_enabling gitops on the provided repo_"? Or so?
```suggestion
		Context("and enabling gitops on the configured repo", func() {
```

File: pkg/nodebootstrap/assets.go (hunk 1 above)
Comment (martina-if) [reply pkg/nodebootstrap/assets.go]: Thanks! I think I still had the old go-bindata locally. Fixed

File: integration/creategetdelete_test.go (hunk 2 above)
Comment (martina-if) [reply integration/creategetdelete_test.go]: yeah... Now I dislike the word "enable" for this :smile: I went with your suggestion. Thanks!

Comment (marccarre) [reply integration/creategetdelete_test.go]: Yes, I don't like it too much either 😅 ... but we said we wouldn't change it so... 🤷‍♂

**Pull request merged**